schunk_gripper_driver/schunk_gripper_driver/tests/test_logging.py

```diff
@@ -39,3 +46,183 @@ def test_driver_doesnt_fill_disk_space_by_default(log_monitor, lifecycle_interfa
         driver.change_state(Transition.TRANSITION_CLEANUP)
 
     # log_monitor does the final asserting
+
+
+@skip_without_gripper
+def test_driver_supports_log_level_changes_via_parameter_call(driver):
```

Comment:
I think this is handled already in `test_parameters.py` where we set all available parameters to some values, right?
If so, I would remove this test here.